content/security/for-admins/group-mapping.md

```diff
@@ -21,6 +21,8 @@ IdPs share with Docker the main attributes of every authorized user through SSO,
 
 Docker uses the email address of the user to identify them on the platform. Every Docker account must have a unique email address at all times.
 
+### SSO authentication with JIT provisioning enabled
```

Comment:
Ah I see what you mean! For now I think we want the disabled info to be pretty prominent since this is a new change. Also wish we could link to specific tabs, because this content is linked from other sections. I am thinking of pulling this into a dedicated provisioning page once this goes to GA and simplify the Group Mapping page.